store/local_cache_supplier_groups.go

```diff
@@ -100,3 +100,13 @@ func (s *LocalCacheSupplier) PendingAutoAddTeamMembers(ctx context.Context, minG
 func (s *LocalCacheSupplier) PendingAutoAddChannelMembers(ctx context.Context, minGroupMembersCreateAt int64, hints ...LayeredStoreHint) *LayeredStoreSupplierResult {
 	return s.Next().PendingAutoAddChannelMembers(ctx, minGroupMembersCreateAt, hints...)
 }
+
+func (s *LocalCacheSupplier) GetGroupsByChannel(ctx context.Context, channelId string, page, perPage int, hints ...LayeredStoreHint) *LayeredStoreSupplierResult {
+	// TODO: Redis caching.
```

Comment:
Some `// TODO: Redis caching.` comments in the wrong file.

Reply:
woops